### ISSUE TITLE
Option to disable updated tabs badge

### ISSUE THREAD

**Issue by Revemohl** [mbnuqw/sidebery#365]
## Steps to reproduce
Open settings, find "Show the mark on unread tabs", set it to off.

## Expected behavior
The blue circle showing if a page is unread should disappear, or at least not show up again in case it's set to off.

## Actual behavior
That blue circle keeps coming back whenever a tab gets updated in the background, even when it's set to off.

## Environment info

OS: win
Firefox: 78.9.0
Addon: 4.9.3

<details><code>

{
  "settings": {
    "version": "4.9.3",
    "nativeScrollbars": false,
    "selWinScreenshots": true,
    "tabsCheck": true,
    "tabsFix": "notify",
    "markWindow": false,
    "markWindowPreface": "[Sidebery] ",
    "ctxMenuNative": false,
    "autoHideCtxMenu": "none",
    "ctxMenuRenderInact": true,
    "ctxMenuIgnoreContainers": "",
    "navBarLayout": "vertical",
    "navBarInline": false,
    "hideAddBtn": false,
    "hideSettingsBtn": true,
    "navBtnCount": true,
    "hideEmptyPanels": true,
    "navMidClickAction": "none",
    "navSwitchPanelsWheel": true,
    "groupLayout": "grid",
    "skipEmptyPanels": false,
    "dndTabAct": true,
    "dndTabActDelay": 750,
    "dndTabActMod": "none",
    "dndExp": "pointer",
    "dndExpDelay": 0,
    "dndExpMod": "none",
    "stateStorage": "session",
    "warnOnMultiTabClose": "collapsed",
    "activateOnMouseUp": true,
    "activateLastTabOnPanelSwitching": true,
    "showTabRmBtn": true,
    "showTabCtx": true,
    "hideInact": false,
    "activateAfterClosing": "next",
    "activateAfterClosingPrevRule": "visible",
    "activateAfterClosingNextRule": "tree",
    "activateAfterClosingGlobal": false,
    "activateAfterClosingNoFolded": true,
    "activateAfterClosingNoDiscarded": true,
    "shiftSelAct": true,
    "askNewBookmarkPlace": false,
    "tabsRmUndoNote": true,
    "nativeHighlight": false,
    "tabsUnreadMark": false,
    "tabsReloadLimit": 5,
    "tabsReloadLimitNotif": true,
    "moveNewTabPin": "end",
    "moveNewTabParent": "last_child",
    "moveNewTabParentActPanel": false,
    "moveNewTab": "end",
    "pinnedTabsPosition": "panel",
    "pinnedTabsList": true,
    "pinnedAutoGroup": false,
    "tabsTree": true,
    "groupOnOpen": true,
    "tabsTreeLimit": "none",
    "hideFoldedTabs": false,
    "autoFoldTabs": false,
    "autoFoldTabsExcept": "none",
    "autoExpandTabs": false,
    "rmChildTabs": "none",
    "tabsChildCount": true,
    "tabsLvlDots": false,
    "discardFolded": false,
    "discardFoldedDelay": 0,
    "discardFoldedDelayUnit": "sec",
    "tabsTreeBookmarks": true,
    "treeRmOutdent": "branch",
    "bookmarksPanel": false,
    "warnOnMultiBookmarkDelete": "collapsed",
    "openBookmarkNewTab": false,
    "midClickBookmark": "open_new_tab",
    "actMidClickTab": false,
    "autoCloseBookmarks": false,
    "autoRemoveOther": false,
    "highlightOpenBookmarks": false,
    "activateOpenBookmarkTab": false,
    "showBookmarkLen": false,
    "bookmarksRmUndoNote": true,
    "fontSize": "xxs",
    "bgNoise": false,
    "animations": true,
    "animationSpeed": "fast",
    "theme": "default",
    "style": "dark",
    "sidebarCSS": false,
    "groupCSS": false,
    "snapNotify": true,
    "snapExcludePrivate": false,
    "snapInterval": 0,
    "snapIntervalUnit": "min",
    "snapLimit": 0,
    "snapLimitUnit": "snap",
    "hScrollThroughPanels": false,
    "scrollThroughTabs": "none",
    "scrollThroughVisibleTabs": false,
    "scrollThroughTabsSkipDiscarded": false,
    "scrollThroughTabsExceptOverflow": true,
    "scrollThroughTabsCyclic": false,
    "tabDoubleClick": "none",
    "tabLongLeftClick": "none",
    "tabLongRightClick": "none",
    "tabsPanelLeftClickAction": "none",
    "tabsPanelDoubleClickAction": "tab",
    "tabsPanelRightClickAction": "menu",
    "tabsPanelMiddleClickAction": "tab",
    "syncName": "",
    "syncSaveSettings": false,
    "syncSaveCtxMenu": false,
    "syncSaveStyles": false,
    "syncAutoApply": false
  },
  "permissions": {
    "allUrls": true,
    "tabHide": true,
    "actualAllUrls": true,
    "actualTabHide": true
  },
  "storage": {
    "size": "359.8 kb",
    "props": {
      "containers_v4": "1 kb",
      "cssVars": "1.6 kb",
      "favAutoCleanTime": "10 b",
      "favUrls": "23.8 kb",
      "favicons": "251.9 kb",
      "lastSnapTime": "13 b",
      "panelIndex": "1 b",
      "panels_v4": "1.7 kb",
      "prevTabsData_v4": "94 b",
      "profileID": "14 b",
      "settings": "2.7 kb",
      "sidebarCSS": "2 b",
      "snapshots_v4": "76.5 kb",
      "tabsData_v4": "94 b"
    }
  },
  "panels": [
    {
      "type": "bookmarks",
      "id": "bookmarks",
      "cookieStoreId": "bookmarks",
      "name": 9,
      "icon": "...",
      "bookmarks": true,
      "lockedPanel": false,
      "skipOnSwitching": false
    },
    {
      "type": "default",
      "id": "firefox-default",
      "name": 7,
      "icon": "...",
      "cookieStoreId": "firefox-default",
      "lockedTabs": false,
      "lockedPanel": false,
      "skipOnSwitching": false,
      "noEmpty": false,
      "newTabCtx": "none",
      "dropTabCtx": "none",
      "moveTabCtx": "none",
      "moveTabCtxNoChild": true
    },
    {
      "type": "tabs",
      "id": "jg1JvIl0hgEp",
      "name": 5,
      "icon": "...",
      "color": "...",
      "customIconSrc": "",
      "customIcon": "",
      "lockedTabs": false,
      "lockedPanel": true,
      "skipOnSwitching": false,
      "noEmpty": true,
      "newTabCtx": "none",
      "dropTabCtx": "none",
      "moveTabCtx": "none",
      "moveTabCtxNoChild": true,
      "urlRulesActive": false,
      "urlRules": ""
    },
    {
      "type": "tabs",
      "id": "AuW7JeDp0GpA",
      "name": 12,
      "icon": "...",
      "color": "...",
      "customIconSrc": "",
      "customIcon": "",
      "lockedTabs": false,
      "lockedPanel": false,
      "skipOnSwitching": false,
      "noEmpty": false,
      "newTabCtx": "none",
      "dropTabCtx": "none",
      "moveTabCtx": "none",
      "moveTabCtxNoChild": true,
      "urlRulesActive": false,
      "urlRules": ""
    },
    {
      "type": "tabs",
      "id": "yGK3I9rJN0pA",
      "name": 3,
      "icon": "...",
      "color": "...",
      "customIconSrc": "",
      "customIcon": "",
      "lockedTabs": false,
      "lockedPanel": false,
      "skipOnSwitching": false,
      "noEmpty": false,
      "newTabCtx": "none",
      "dropTabCtx": "none",
      "moveTabCtx": "none",
      "moveTabCtxNoChild": true,
      "urlRulesActive": false,
      "urlRules": ""
    },
    {
      "type": "tabs",
      "id": "w5lp-gtOecqA",
      "name": 7,
      "icon": "...",
      "color": "...",
      "customIconSrc": "",
      "customIcon": "",
      "lockedTabs": false,
      "lockedPanel": false,
      "skipOnSwitching": false,
      "noEmpty": false,
      "newTabCtx": "none",
      "dropTabCtx": "none",
      "moveTabCtx": "none",
      "moveTabCtxNoChild": true,
      "urlRulesActive": false,
      "urlRules": ""
    }
  ],
  "containers": [
    {
      "id": "firefox-container-1",
      "name": 8,
      "icon": "...",
      "color": "...",
      "proxified": false,
      "proxy": null,
      "includeHostsActive": false,
      "includeHosts": "",
      "excludeHostsActive": false,
      "excludeHosts": "",
      "userAgentActive": false,
      "userAgent": ""
    },
    {
      "id": "firefox-container-2",
      "name": 4,
      "icon": "...",
      "color": "...",
      "proxified": false,
      "proxy": null,
      "includeHostsActive": false,
      "includeHosts": "",
      "excludeHostsActive": false,
      "excludeHosts": "",
      "userAgentActive": false,
      "userAgent": ""
    },
    {
      "id": "firefox-container-3",
      "name": 7,
      "icon": "...",
      "color": "...",
      "proxified": false,
      "proxy": null,
      "includeHostsActive": false,
      "includeHosts": "",
      "excludeHostsActive": false,
      "excludeHosts": "",
      "userAgentActive": false,
      "userAgent": ""
    },
    {
      "id": "firefox-container-4",
      "name": 8,
      "icon": "...",
      "color": "...",
      "proxified": false,
      "proxy": null,
      "includeHostsActive": false,
      "includeHosts": "",
      "excludeHostsActive": false,
      "excludeHosts": "",
      "userAgentActive": false,
      "userAgent": ""
    }
  ],
  "cssVars": {
    "bg": "#24262F",
    "tabs_activated_bg": "#1E1D26"
  },
  "sidebarCSSLen": 0,
  "windows": [
    {
      "state": "maximized",
      "incognito": false,
      "tabsCount": 386
    }
  ],
  "bookmarks": {
    "bookmarksCount": 1079,
    "foldersCount": 19,
    "separatorsCount": 0,
    "maxDepth": 4
  }
}
</code></details>

**Comment by mbnuqw** [comment]
Unread tab's mark is a turquoise triangle at the bottom-left.
![screenshot-2021-03-27 19:54:18](https://user-images.githubusercontent.com/6276694/112724664-47c48700-8f36-11eb-9325-e65651750469.png)

The blue circle badge shows that tab's title was changed (useful for messaging / streaming sites).
![screenshot-2021-03-27 19:54:57](https://user-images.githubusercontent.com/6276694/112724678-54e17600-8f36-11eb-983a-01f665d1b00e.png)

Unfortunately, there is no toggle for this feature and if you don't mind I can update this issue as "Option to disable updated tabs badge" and implement it later.

As a workaround you can try this custom css (Sidebery settings / Appearance / Edit styles):
```css
.Tab[data-updated="true"] .fav .update-badge {
  display: none;
}

.Tab[data-updated="true"] .fav img,
.Tab[data-updated="true"] .fav svg {
  mask: none;
}
```

**Comment by Revemohl** [comment]
My bad, please feel free to change the issue then and thanks for the workaround.

**Comment by mbnuqw** [comment]
Available from v5.0.0
